package.json

```diff
@@ -546,6 +546,87 @@
         }
       }
     ],
+    "walkthroughs": [
+      {
+        "id": "TektonWalkthrough",
+        "title": "Getting Started with Tekton Pipeline",
+        "description": "Start creating, running, and managing your Tekton pipelines on OpenShift and Kubernetes",
+        "steps": [
+          {
+            "id": "Create Resources",
+            "title": "Create new resource",
+            "description": "The plugin provide a smart yaml-code completion support which make it easy to create any Tekton resource by using the IDE editor. To create new resource open new file in editor and save file <name>.yaml and once you have done editing it on save you can deploy the resource.",
+            "media": {
+              "image": "https://raw.githubusercontent.com/wiki/redhat-developer/vscode-tekton/images/walkthroughs/createPipeline.gif",
+              "altText": "create Resources"
+            }
+          },
+          {
+            "id": "Start a pipeline/task",
+            "title": "Start a pipeline/task",
+            "description": "After having created/edited your pipeline/task and successfully deployed it on cluster, you can run it by executing the `Start Pipeline/Task` action.\n A wizard will pop up leading you through the process of setting all inputs necessary to get it started (params, workspaces, service account, ..).\n Note: By default if the pipeline/task has no inputs, it will get started directly without the wizard popping up.\n If you already run a pipeline/task earlier and wants to run it again with the same inputs you did last time, you can achieve it by executing the `Start Last Run` action from the Pipelines/Tasks context menu nodes.\n Opposite, if you want to start a pipeline/task with the same inputs of a specific old run (not the latest), you can do it by selecting the old pipelinerun/taskrun node and execute the `Restart` action. This will open up the wizard pre-filled with the inputs used to start that pipeline/task in the past",
```

Comment:
```
If you already run a pipeline/task earlier and wants to run it again with the same inputs you did last time, you can 
achieve it by executing the `Start Last Run` action from the Pipelines/Tasks context menu nodes.\n Opposite, if you 
want to start a pipeline/task with the same inputs of a specific old run (not the latest), you can do it by selecting the old 
pipelinerun/taskrun node and execute the `Restart` action. This will open up the wizard pre-filled with the inputs used to
 start that pipeline/task in the past"
```
The last part of this description is not correct for vscode. In IJ we have the `start last run` and `mirror start` actions for both pipeline/run and task/run but in vscode is different. I can see the `Start last run` action enabled for pipelines and not for tasks and the `Restart` action for a taskrun does not open up the webview. I think better to only mention pipeline and removed task from this section or rewrite it more clearly.